stdpopsim/arabidopsis_thaliana.py

```diff
@@ -7,7 +7,7 @@
 import stdpopsim.models as models
 import stdpopsim.genomes as genomes
 import stdpopsim.genetic_maps as genetic_maps
-
+import stdpopsim.generic_models as generics # NOQA
```

Comment:
Doesn't  this mean that we're defining exactly the same models (with no modifications) multiple times just with different paths? This seems like a bad thing to me. What's wrong with defining a generic population model and letting users set the generation time, etc, themselves using the values defined in the various species modules? I feel like we're over-designing this.